### DISCUSSION TITLE
Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.

### DISCUSSION THREAD

**Discussion by Pabl0Parra** [prisma/prisma#15914] [Q&A] [answered]
Hello,

I´m trying to run the backend of a project, when I run "npm run backend" I get the above error message. Anyone has any idea of why?

```
PS C:\Users\pparr\PROYECTO IT ACADEMY\ita-directory> npm run backend

> ita-directory@1.0.0 backend
> cp .env.development ./backend/.env && cd ./backend && npm run dev


> be_ita_directory@1.0.0 dev
> set NODE_ENV=development && nodemon --legacy-watch server.js

[nodemon] 2.0.7
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,json
[nodemon] starting `node server.js`
C:\Users\pparr\PROYECTO IT ACADEMY\ita-directory\backend\node_modules\.prisma\client\index.js:3
    throw new Error(
    ^

Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues
    at new PrismaClient (C:\Users\pparr\PROYECTO IT ACADEMY\ita-directory\backend\node_modules\.prisma\client\index.js:3:11)
    at Object.<anonymous> (C:\Users\pparr\PROYECTO IT ACADEMY\ita-directory\backend\prisma\indexPrisma.js:5:16)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (C:\Users\pparr\PROYECTO IT ACADEMY\ita-directory\backend\app\controllers\users.controller.js:12:16)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by austincrim** [comment, marked as answer]
Hey @Pabl0Parra 👋,

Have you tried running `npx prisma generate` per the error message?

**Reply by Pabl0Parra** [reply]
Hello, I was able to fix that part, running `npx prisma generate`, but now I get the following error
![last_error](https://user-images.githubusercontent.com/98957023/197581519-4ec9ed9b-9c03-4d50-9107-1f70a8dfbc29.PNG)

**Reply by janpio** [reply]
You should upgrade to a newer Prisma version - that would definitely give a better error message here, but possibly also completely avoid the error already.

**Reply by Pabl0Parra** [reply]
thanks sir, I was able to get rid of the error by installing openSSL 3.1, after that I did upgrade to the newer Prisma version

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
